Blaise-Shyaka/My-Mentor
# [finishes #170380194] finish view specific mentor page

# What does this PR do?

This page allows a user to view a specific mentor

# Description of tasks to be completed

- The page should be responsive
- It should contain the mentor's personal details [full name, address, and occupation], bio, and expertise
- It should allow the user to go back to their dashboard or go back to view all mentors

# How should this be manually tested?

- Switch to the views directory and open individual-mentor.html
- Resize the browser to verify the page responsiveness
- Check on the left of the page a "view all mentors" button 
- Check on the right of the page a "my dashboard" link 
- Check the presence of the mentor's full name, address, occupation, bio, and expertise on the page
